scripts/pi-hole/js/footer.js

```diff
@@ -73,19 +117,31 @@ $("#pihole-disable-permanently").on("click", function(e){
 $("#pihole-disable-10s").on("click", function(e){
     e.preventDefault();
     piholeChange("disable","10");
-    setTimeout(function(){piholeChanged("enabled");},10000);
+    setCountdownTarget(10);
+    setTimeout(countDown,1000);
```

Comment:
You could integrate those two lines into `piholeChange()` so they don't have to be repeated over and over (see alsi lines 126-127, 132-133, 140-141)